functions/UpdateBookingSlot-BookToISV/run.csx

```diff
@@ -104,7 +103,7 @@ public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, CloudT
             // Compile and Write the mail record away to the ServiceBus if this is enabled for this TE (if the TE master record is populated);
 
             // Find the TE master data record 
-            technicalevangelist chkte = (from te in tblte.CreateQuery<technicalevangelist>() select te).Where(e => e.RowKey == bsout.TechnicalEvangelist && e.PartitionKey == "ALL").FirstOrDefault();
+            TechnicalResource chkte = (from te in tblte.CreateQuery<TechnicalResource>() select te).Where(e => e.RowKey == bsout.TechnicalEvangelist && e.PartitionKey == "ALL").FirstOrDefault();
```

Comment:
Renamed it as it was very specific to our role. If this is a public booking system, then people wouldn't get the 'evangelist' bit. I also thought in the future, that maybe you could get CSA on here. If I was going to change the class type, then I should have done a full job - sloppy! Sorry!